client/src/components/ElementEditor/FormBuilder.scss

```diff
@@ -1,3 +1,7 @@
 .element-editor-formbuilder {
   margin-top: $spacer-xs;
+  cursor: auto;
+}
+.element-editor-formbuilder--collapsed {
```

Comment:
We discussed recently (I think while you were on leave) that we prefer to nest modifiers, e.g.

```scss
.element-edit-formbuilder {
  // ...
  &--collapsed {
    display: none;
  }
}
```

Reply:
Seems reasonable :)